connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedConfig.java

```diff
@@ -193,15 +192,6 @@
     public static final String INTER_WORKER_VERIFICATION_ALGORITHMS_DOC = "A list of permitted algorithms for verifying internal requests";
     public static final List<String> INTER_WORKER_VERIFICATION_ALGORITHMS_DEFAULT = Collections.singletonList(INTER_WORKER_SIGNATURE_ALGORITHM_DEFAULT);
 
-    private static final Validator REPLICATION_FACTOR_VALIDATOR = LambdaValidator.with(
```

Comment:
These had to be moved out of this class, or else tests for `StandaloneHerder` would break. They are not reused here as well as `TopicCreationConfig`